get_dependencies.py

```diff
@@ -26,17 +27,14 @@ def download(url, filename):
 
 if sys.platform.startswith('linux'):
     url = 'https://www.dropbox.com/s/ziti4nkdzhgwg1n/linux.tar.xz?dl=1'
-    name = 'linux.tar.xz'
 elif sys.platform.startswith('darwin'):
     url = 'https://www.dropbox.com/s/k4yifantsypy9xv/mac.tar.xz?dl=1'
-    name = 'mac.tar.xz'
 elif sys.platform.startswith('win32'):
     url = 'https://www.dropbox.com/s/xskj9rpn2fjkra8/win32.tar.xz?dl=1'
-    name = 'win32.tar.xz'
 
 print('[*] Downloading support files...')
-download(url, name)
-
+download(url)
+name = url[url.rfind("/") + 1:url.find("?")]
```

Comment:
Make `download` return the name, so we avoid duplicating the code here.

Reply:
fname = download(url)
 tarfile.open(fname, 'r:xz')
....
os.remove(fname)
may i use this way if download return the filename?

Reply:
Why do you want to change the name of the variable from `name` to `fname`?

Reply:
variable will be name only , then is this permissible  assigning download(url) to name?

Reply:
apologies me , but just asking that download(url) will download file though it will return filename yet assigning to a variable named 'name' may cause to misunderstand the variable (for the new ones).

Reply:
I still don't understand the question 😄 
Let's leave the name of the variable as `name`, we can consider changing it in the future in a separate PR.

Reply:
okay